opacus/tests/dp_layers/dp_rnn_test.py

```diff
@@ -43,12 +44,15 @@ class DPLSTM_test(DPModules_test):
         batch_first=st.booleans(),
         zero_init=st.booleans(),
         packed_input_flag=st.integers(
-            0, 2
-        ),  # 0 indicates no packed sequence input, 1 indicates packed sequence input in sorted order, 2 indicates packed sequence input in unsorted order
+            0,  # no packed sequence input
+            # 1,  # packed sequence input in sorted order
```

Comment:
noob: why is this not tested?

Reply:
oh
it it considered, but the comment is extremely misleading. The change is again coming from #222 

`st.integers(0,2)` actually sets a range -> it iterates over [0,1,2]
The intention of this comment is to provide an explanation for each input flag value, while integrated into the code. But I think it failed to do so, as it looks obviously as if we're skipping `1`.
I'll revert to old comment

Reply:
My bad, misunderstood the meaning.